lib/envs.js

```diff
@@ -138,6 +139,20 @@ function loadEnvs(path) {
   return env;
 }
 
+function loadEnvs(path) {
+  if (path.indexOf(',') === -1) {
+    return loadEnvsFile(path);
+  }
+
+  // Load multiple ENV files
+  var paths = path.split(',');
```

Comment:
I'm not usually a fan of overly terse code, but in this case.. I think the shorter form _might_ be more readable at a glance.

```js
return path.split(',').map(loadEnvsFile).reduce(util._extend, {});
```